Code/GraphMol/SynthonSpaceSearch/SearchResults.h

```diff
@@ -0,0 +1,68 @@
+//
+// Copyright (C) David Cosgrove 2024.
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+#include <RDGeneral/export.h>
+#ifndef RDKIT_SYNTHONSPACE_SEARCHRESULTS_H
+#define RDKIT_SYNTHONSPACE_SEARCHRESULTS_H
+
+#include <GraphMol/ROMol.h>
+
+namespace RDKit::SynthonSpaceSearch {
+class RDKIT_SYNTHONSPACESEARCH_EXPORT SearchResults {
+ public:
+  explicit SearchResults() : d_maxNumResults(0) {}
+  SearchResults(std::vector<std::unique_ptr<ROMol>> &&mols, size_t maxNumRes);
+  SearchResults(const SearchResults &other);
+  SearchResults(SearchResults &&other) = default;
+  ~SearchResults() = default;
+
+  SearchResults &operator=(const SearchResults &other);
+  SearchResults &operator=(SearchResults &&other) = default;
+
```

Comment:
That breaks the python wrapping.  Probably having SynthonSpaceSearcher::search() return a pointer to SearchResults rather than a copy of it would be better, but I couldn't wrap that, either.

Reply:
I briefly played around with it and also see the problem.
I think we can fix it, but it can be done in the next iteration